package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "amazon-connect-chatjs",
```

Comment:
Instead of a patch release , lets do a minor release ?
1.4.0

Reply:
Good catch, will publish rev now.